### PR TITLE
Add a stop hook to clean session(start milvus as a child process)

### PR DESCRIPTION
**Pull request by jaime0815**
issue: https://github.com/milvus-io/milvus/issues/27270
pr: #27564

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -17,11 +17,59 @@
...
+			roles := milvus.GetMilvusRoles(args, flag.NewFlagSet(args[0], flag.ExitOnError))
+			metaPath := params.EtcdCfg.MetaRootPath
+			endpoints := params.EtcdCfg.Endpoints
+
+			sessionSuffix := sessionutil.GetSessions(cmd.Process.Pid)
+			defer sessionutil.RemoveServerInfoFile(cmd.Process.Pid)
+
+			// clean session
+			if err := milvus.CleanSession(metaPath, endpoints, sessionSuffix); err != nil {
+				fmt.Println("clean session failed, server type =", roles)
```
Comment (xiaofan-luan) [inline cmd/main.go:63]: use log.Println?

Comment (xiaofan-luan) [reply cmd/main.go:63]: print the error?

File: scripts/start_standalone.sh [hunk 2]
```diff
@@ -28,4 +28,4 @@ fi
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$PWD/internal/core/output/lib/
 echo "Starting standalone..."
-nohup ./bin/milvus run standalone > /tmp/standalone.log 2>&1 &
+nohup ./bin/milvus run standalone --run-with-subprocess > /tmp/standalone.log 2>&1 &
```
Comment (xiaofan-luan) [inline scripts/start_standalone.sh:31]: why this valid only for standalone but not for start_cluster.sh

File: cmd/main.go [hunk 3]
```diff
@@ -17,11 +17,59 @@
...
+		fmt.Println("run subprocess with cmd:", args)
+
+		/* #nosec G204 */
+		cmd := exec.Command(args[0], args[1:]...)
+
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+
+		// No need to extra wait for the process
+		err := cmd.Run()
```
Comment (xiaofan-luan) [inline cmd/main.go:48]: this will block until server crash?

Comment (jaime0815) [reply cmd/main.go:48]: Yes,   it will block until the server exits with normal or abnormal status.

**Pull request merged**